Kitodo/src/main/java/org/kitodo/production/forms/ProzesskopieForm.java

```diff
@@ -498,7 +511,31 @@ protected void removeCollectionsForChildren(LegacyMetsModsDigitalDocumentHelper
      * Create the process and save the metadata.
      */
     public String createNewProcess() {
+        if (Objects.nonNull(titleRecordLinkTab.getTitleRecordProcess())) {
+            if (Objects.isNull(titleRecordLinkTab.getInsertionPositionSelectionSelectedItem())) {
+                Helper.setErrorMessage("prozesskopieForm.createNewProcess.noInsertionPositionSelected");
+                return stayOnCurrentPage;
+            } else {
+                User titleRecordOpenUser = DataEditorForm
+                        .getUserOpened(titleRecordLinkTab.getTitleRecordProcess().getId());
+                if (Objects.nonNull(titleRecordOpenUser)) {
+                    Helper.setErrorMessage("prozesskopieForm.createNewProcess.titleRecordOpen",
+                        titleRecordOpenUser.getFullName());
+                    return stayOnCurrentPage;
```

Comment:
If a user has just opened the same process and then saves it, he would overwrite the changed file without his changes being "merged" into it. That is, the newly inserted link would be deleted. It is extremely rare that this could happen. In this case, blocking the creation of the child process and displaying the user editing the parent process so that the two operators can discuss it seems to me a first, simple solution.